frontend/components/DataFldDesc.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <div class="fieldContainer">
+    <div class="companyIcon">
+      <img
+        class="imgIcon"
+        src="../assets/blumilklogo.png"
+        alt="Company Logo Icon"
+      />
+    </div>
+    <div class="companyText">
+      <span>Nazwa Bardzo Duzej Firmy, w sumie to nawet jeszcze większej</span>
+    </div>
+    <div class="course noselect">
+      <span class="noselect">Informatyka</span>
+    </div>
+    <div class="tagsList textContainer noselect">
+      <span class="tags">.Net,C#,Azure,JS, tag1, tag2, tag3</span>
+    </div>
+    <div class="city"><span>Legnica</span></div>
+    <div class="payInfo">
+      <img src="../assets/paidicon.svg" alt="paidicon" />
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {};
+  },
+};
+</script>
+
+<style lang="scss" scoped>
```

Comment:
Couldn't you use tailwind for those styles?

Reply:
Yeah, we were not familiar with Tailwind styling for now. We will try to change into it later on.